modules/ROOT/pages/setup.adoc

```diff
@@ -95,41 +95,32 @@ Anypoint Partner Manager enables you to connect your own storage solution (such
 You'll need to establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
 The follow the steps explain how to setup the Payload Storage API:
 
-. Download the Partner Manager Content Storage API and sample implementation template from Anypoint Exchange.
-Install this API on a Mule runtime that can access the storage solution.**[MISSING INFO - Confirm final name by GA. Add link if it's live before GA.]**
+. Download the *Partner Manager Content Storage API* and sample implementation template from Anypoint Exchange.
+Install this API on a Mule runtime that can access the storage solution.
 
-. Make a note of the API address configurations (make a note of the following):
+. Make a note of the API address configurations, for example:
 
-** host, port, path connection
+* host, port, path connection
 
-** response timeouts
+* response timeouts
 
-** authentication details: None, API Key, Basic, Client Credentials, OAuth Token
+* authentication types: None, API Key, Basic, Client Credentials, OAuth Token
 
-** Certificate (.PEM format) and connection type
+* Certificate (`.PEM` format) and connection type
 
-[xref] to HTTPS Send Configuration for complete details?
-
-You'll need these details while associating Anypoint Partner Manager to your Payload Storage API. +
+You'll need this information while associating Anypoint Partner Manager to your Payload Storage API. +
 
 NOTE: Your payload storage API must be accessible over the internet via HTTPS.
```

Comment:
This sounds like it should be a task prerequisite, mentioned before Step 1.

Also...this text lowercases "payload storage API," whereas the previous text capitalizes it like a proper noun. Which is correct? Fix the rest.

Reply:
Itt's capped where the user will download it. I htink it's lowercase here.